src/flows/components/header/index.tsx

```diff
@@ -145,6 +152,21 @@ const FlowHeader: FC = () => {
     /* eslint-enable no-console */
   }
 
+  const handleClone = async () => {
+    event('clone_notebook')
+    const clonedId = await clone(flow.id)
+    history.push(
+      `/orgs/${orgID}/${PROJECT_NAME_PLURAL.toLowerCase()}/${clonedId}`
+    )
+  }
+
+  const handleDelete = () => {
```

Comment:
Question to ponder on about. Do we want to send an event to remove the data from redis? What’s the user story here for the other person on this notebook?

Reply:
we def will need to handle that case